packages/amplify-graphql-sql-transformer/src/__tests__/__snapshots__/amplify-graphql-sql-transformer.test.ts.snap

```diff
@@ -1,5 +1,53 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`sql directive tests Post auth resolver Renders post auth resolver if sandbox is disabled and iam access disabled 1`] = `
+"## [Start] Sandbox Mode Disabled. **
+#if( !$ctx.stash.get(\\"hasAuth\\") )
+  $util.unauthorized()
+#end
+$util.toJson({})
+## [End] Sandbox Mode Disabled. **"
+`;
+
+exports[`sql directive tests Post auth resolver Renders post auth resolver if sandbox is disabled and iam access enabled 1`] = `
+"## [Start] Sandbox Mode Disabled. **
```

Comment:
nit: readability comment - This resolver snippet b/w [Start] and [End] depends on both sandbox mode and iam access states. Making this `## [Start] Sandbox Mode Disabled and IAM access Enabled **` would make it more clear.

Reply:
I'll make it part of renaming PR discussed above https://github.com/aws-amplify/amplify-category-api/pull/2398#discussion_r1543340299 .